inbox/mailsync/backends/imap/common.py

```diff
@@ -204,6 +210,10 @@ def remove_deleted_uids(account_id, folder_id, uids):
                     ImapUid.folder_id == folder_id,
                     ImapUid.msg_uid == uid,
                 )
+                .with_hint(
+                    ImapUid,
+                    "FORCE INDEX (ix_imapuid_account_id_folder_id_msg_uid_desc)",
+                )
```

Comment:
Not a covering index since we retrieve other columns, but does not make anything slower, just switches the index.